extensions/MozChangeField/web/js/set-tracking-severity-s1.js

```diff
@@ -0,0 +1,102 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0. */
+
+/**
+ * Reference or define the Bugzilla app namespace.
+ * @namespace
+ */
+var Bugzilla = Bugzilla || {}; // eslint-disable-line no-var
+
+/**
+ * Enforce setting of Priority to P1 when Severity is set to S1
+ */
+Bugzilla.SetTrackingSeverityS1 = class SetTrackingSeverityS1 {
+  /**
+   * Initialize a new SetTrackingSeverityS1 instance.
+   */
+  constructor() {
+    this.priority = document.querySelector("#priority");
+    this.severity = document.querySelector("#bug_severity");
+    this.flags = document.querySelector("div.edit-show table.tracking-flags");
+    this.firefox_versions = document.querySelector(
+      'meta[name="firefox-versions"]'
+    );
```

Comment:
Can you check that `[% IF firefox_versions %]` in Template Toolkit returns false if `firefox_versions` is an empty hashref (Perl returns true in this case)?